src/test/objectstore/test_bluefs.cc

```diff
@@ -642,6 +642,32 @@ TEST(BlueFS, test_compaction_sync) {
     writes_done = true;
     join_all(sync_threads);
     fs.compact_log();
+
+    {
+      string dir = "dir.after_compact_test";
+      ASSERT_EQ(0, fs.mkdir(dir));
+      string file = "file.after_compact_test";
```

Comment:
File and Directory and data are all accessed outside this scope, move their names outside the scope instead of redefining them.
(nitpick - change variables names from dir and file to dir_name and file_name)

Reply:
done